packages/@webex/plugin-meetings/src/reconnection-manager/index.ts

```diff
@@ -570,9 +571,11 @@ export default class ReconnectionManager {
 
     // resend media requests
     if (this.meeting.isMultistream) {
-      Object.values(this.meeting.mediaRequestManagers).forEach((mediaRequestManager) =>
-        // @ts-ignore - Fix type
-        mediaRequestManager.commit()
+      Object.values(this.meeting.mediaRequestManagers).forEach(
+        (mediaRequestManager: MediaRequestManager) => {
+          mediaRequestManager.clearPreviousRequests();
```

Comment:
is there maybe some existing test that could be updated with the check that clearPreviousRequests() is called?

Reply:
Yeah, I'll add it to the unit tests for MediaRequestManager.